run_dir/static/js/running_notes.js

```diff
@@ -179,7 +209,19 @@ $(document).ready(function(){
       $('[data-toggle="tooltip"]').click(function (){
          $('[data-toggle="tooltip"]').tooltip("hide");
       });
-})
+      var cat_cards = [];
+      $('#running_notes_panels').find('.badge').each(function(){
+          var label = $.trim($(this).text())
+          if (label){
+              if($.inArray(label, cat_cards) !== -1){
+                  cat_cards[label]++;
+              }
+              else{
+                  cat_cards[label] = 1;
+              }
+          }
```

Comment:
Does this do anything?

Reply:
Yes 🙈 that's what working with screen sharing does 🤦‍♀️